docs/sources/tutorials/get-started.md

```diff
@@ -71,17 +86,161 @@ services:
           editable: false
         EOF
         /run.sh
-    image: grafana/grafana:latest
+    image: grafana/grafana:11.0.0
     ports:
       - "3000:3000"
 ```
 
-After running `docker-compose up`, open [http://localhost:3000](http://localhost:3000) in your browser to view the Grafana UI.
+After running `docker-compose up`, open [http://localhost:3000](http://localhost:3000) in your browser to view the Grafana UI. In some newer
+docker installs, this command may be run as `docker compose up` without the dash character.
```

Comment:
your question is reasonable and has answers but I really really don't want to answer it because the answer is different per OS.  It's much easier to be like, "Just try both commands".  To make matters worse, newer installs per OS (which expect "docker compose") will work with "docker-compose" but others won't.

Reply:
I did some digging and it seems like docker-compose is a deprecated v1... and docker compose is the newer v2.  https://docs.docker.com/compose/migrate/  

So.. my suggestion is to use the "current" v2 command as the primary step content... and offer `docker-compose` as an alternative if `docker compose` fails.